CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 See [Upgrading] for details on how to upgrade.
 
+- Fix bug stored xss - Data when render on FE allows execution of arbitrary javascript code
```

Comment:
```suggestion
- Fix Multiple Stored XSS in Administration allowing execution of arbitrary JavaScript code, #1260
```

1. update changelog to make sense to the end user
2. add PR link

also, update PR title to match changelog entry